### PR TITLE
Remove references to Stopwatch

### PR DESCRIPTION
**Pull request by Bucimis**
This will help us develop Stopwatch more efficiently and future proof for future changes/migrations.

In the meantime, as we encounter customer confusion, we should provide embedded examples.

### CODE REVIEW

**Review by chshapiro — APPROVED** [review]

File: _docs/_developer_guide/platform_integration_guides/ios/push_notifications/customization.md [hunk 1]
```diff
@@ -34,9 +34,7 @@ Please note that setting the badge number to 0 will also clear up notifications
 
 In addition to providing a set of [default push categories][2], Braze supports custom notification categories and actions. Once you register categories in your application, you can use the Braze dashboard to send notification categories to your users.
 
-For an example of setting up custom actions and categories, please see the `setupRemoteNotificationForiOS10` or `setupRemoteNotificationForiOS8And9` methods within the [AppDelegate.m][15] file of Stopwatch.
-
->  If you haven't migrated to iOS 10, please see this [alternative categories documentation][31].
+>  If are not using the `UserNotifications` framework, please see this [alternative categories documentation][31].
```
Comment (mattlsteinberg) [inline _docs/_developer_guide/platform_integration_guides/ios/push_notifications/customization.md:37]:
```suggestion
>  If you are not using the `UserNotifications` framework, please see this [alternative categories documentation][31].
```

**Review by mattlsteinberg — APPROVED** [review]
One nit but LGTM!

**Comment by KellieHawks** [comment]
@Bucimis Looks good to me. Seems like a couple more people might review it, so please tag me when you are ready to merge. Thanks!

**Comment by Bucimis** [comment]
@KellieHawks ready :)

**Review by hokstuff — APPROVED** [review]

**Pull request merged**